src/main/java/bookstore/controller/BookController.java

```diff
@@ -25,11 +29,23 @@ public List<BookDto> getAll() {
 
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable Long id) {
-        return bookService.findBookById(id);
+        return bookService.findById(id);
     }
 
     @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
     public BookDto createBook(@RequestBody CreateBookRequestDto book) {
         return bookService.save(book);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBookById(@PathVariable Long id, @RequestBody CreateBookRequestDto book) {
```

Comment:
you may also validate such things as `@PathVariable, @RequestParam` etc:
```
@PathVariable @Positive Long id
```

in order to do so you need to annotate your controller with `@Validated` annotation

Reply:
Validation is one topic ahead in course and just next homework